src/Components/v2/AuctionCardLuxon.tsx

```diff
@@ -0,0 +1,191 @@
+import {
+  BorderBox,
+  Box,
+  Flex,
+  Image,
+  Link,
+  ResponsiveImage,
+  Sans,
+  Serif,
+} from "@artsy/palette"
+import { AuctionCard_sale } from "__generated__/AuctionCard_sale.graphql"
+import { Truncator } from "Components/Truncator"
+import { DateTime } from "luxon"
+// import moment from "moment-timezone"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { get } from "Utils/get"
+import { Media } from "Utils/Responsive"
+
+export const relativeTime = (timeIn, now) => {
+  const time = DateTime.fromISO(timeIn)
+  const abs = Math.abs
+  if (abs(time.diff(now).as("days")) >= 1) {
+    return `${Math.floor(time.diff(now).as("days"))}d`
+  } else if (abs(time.diff(now).as("hours")) >= 1) {
+    return `${Math.floor(time.diff(now).as("hours"))}h`
+  } else if (abs(time.diff(now).as("minutes")) >= 1) {
+    return `${Math.floor(time.diff(now).as("minutes"))}m`
+  }
+  return `${Math.floor(time.diff(now).as("seconds") % 60)}s`
+}
+
+// now defaults to DateTime.local() but can be overriden for unit testing
+export const upcomingLabel = (sale, now = DateTime.local()) => {
+  const {
+    start_at: startAt,
+    end_at: endAt,
+    live_start_at: liveStartAt,
+    is_closed: isClosed,
+    is_live_open: isLiveOpen,
+    is_preview: isPreview,
+    registration_status,
+    is_registration_closed: isRegistrationClosed,
+  } = sale
+
+  const isRegistered = !!registration_status
+  const isLAI = !!liveStartAt
+  if (isPreview) {
+    return `Opens in ${relativeTime(startAt, now)}`
+  } else if (isClosed) {
+    return "Auction closed"
+  } else if (isLAI) {
+    if (isLiveOpen) {
+      return "In progress"
+    } else if (isRegistered || isRegistrationClosed) {
+      return `Live in ${relativeTime(liveStartAt, now)}`
+    } else {
+      const dateTime = DateTime.fromISO(liveStartAt).setZone("America/New_York")
+      return `Register by ${dateTime.monthShort} ${dateTime.day}`
+    }
+  } else {
+    return `Ends in ${relativeTime(endAt, now)}`
+  }
```

Comment:
One thing to note is a lot of similar behavior is already in MP: https://github.com/damassi/metaphysics/blob/d6b855cb928e51b8c8b298086540cd3b9734fa04/src/schema/sale/display.ts#L34